src/common/DataProvider/SampleDataProvider.tsx

```diff
@@ -5,6 +5,7 @@
 import { DelayLoadedTreeNodeItem, ITreeDataProvider, TreeDataChangesListener, TreeNodeItem } from "@bentley/ui-components";
 import { PropertyRecord } from "@bentley/ui-abstract";
 import { BeEvent } from "@bentley/bentleyjs-core";
+import "./Trees.scss";
```

Comment:
Is this needed?

Reply:
This stylesheet needs to be referenced somewhere once to be discoverable by webpack in order to get it bundled with the webapp.